client/state/jetpack-connect/selectors.js

```diff
@@ -103,33 +102,32 @@ const getUserAlreadyConnected = state => {
 };
 
 /**
- * XMLRPC errors can be identified by the presence of an error message, the presence of an authorization code
```

Comment:
That previous comment was really informative. I'd prefer keeping it below the one you suggested.

Reply:
I changed it because it seemed to describe, in detail, the internals of the function, which I tend to avoid.. I'll add it back.

Reply:
>  keeping it below

Missed this. 👍  for adding a detailed description.